openqabot/approver.py

```diff
@@ -143,6 +155,7 @@ def is_job_marked_acceptable_for_incident(self, job_id: int, inc: int) -> bool:
             for comment in self.client.get_job_comments(job_id):
                 sanitized_text = sanitize_comment_text(comment["text"])
                 if regex.search(sanitized_text):
+                    self.mark_job_as_acceptable_for_incident(job_id, inc)
```

Comment:
It reads a bit weirdly that the function "is job marked acceptable" calls "mark job as acceptable". I suggest to move this line to https://github.com/openSUSE/qem-bot/pull/191/files#diff-d44cab80bf43c62865043529f6e9883cc12ab37d49f5fb5af9e1bdc7c012dcaaR288